[2024-04-15T16:22:39.370+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-15T16:22:39.411+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_airflow_dag.echo_hello_airflow scheduled__2024-04-14T00:00:00+00:00 [queued]>
[2024-04-15T16:22:39.423+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_airflow_dag.echo_hello_airflow scheduled__2024-04-14T00:00:00+00:00 [queued]>
[2024-04-15T16:22:39.424+0000] {taskinstance.py:2303} INFO - Starting attempt 3 of 8
[2024-04-15T16:22:39.445+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): echo_hello_airflow> on 2024-04-14 00:00:00+00:00
[2024-04-15T16:22:39.459+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=148) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-15T16:22:39.461+0000] {standard_task_runner.py:63} INFO - Started process 166 to run task
[2024-04-15T16:22:39.460+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hello_***_dag', 'echo_hello_***', 'scheduled__2024-04-14T00:00:00+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/hello_***_dag.py', '--cfg-path', '/tmp/tmp9uiacj5i']
[2024-04-15T16:22:39.478+0000] {standard_task_runner.py:91} INFO - Job 52: Subtask echo_hello_***
[2024-04-15T16:22:39.550+0000] {task_command.py:426} INFO - Running <TaskInstance: hello_airflow_dag.echo_hello_airflow scheduled__2024-04-14T00:00:00+00:00 [running]> on host ead826da457c
[2024-04-15T16:22:39.674+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='yazid' AIRFLOW_CTX_DAG_ID='hello_***_dag' AIRFLOW_CTX_TASK_ID='echo_hello_***' AIRFLOW_CTX_EXECUTION_DATE='2024-04-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-14T00:00:00+00:00'
[2024-04-15T16:22:39.676+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-15T16:22:39.678+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-15T16:22:39.679+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello ***']
[2024-04-15T16:22:39.702+0000] {subprocess.py:86} INFO - Output:
[2024-04-15T16:22:39.723+0000] {subprocess.py:93} INFO - hello ***
[2024-04-15T16:22:39.726+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-15T16:22:39.727+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-15T16:22:39.758+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=hello_***_dag, task_id=echo_hello_***, execution_date=20240414T000000, start_date=20240415T162239, end_date=20240415T162239
[2024-04-15T16:22:39.799+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-15T16:22:39.853+0000] {taskinstance.py:3482} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-04-15T16:22:39.862+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
